packages/agent/src/polling/index.ts

```diff
@@ -16,52 +14,6 @@ export type PollStrategy = (
 ) => Promise<void>;
 export type PollStrategyFactory = () => PollStrategy;
 
-interface SignedReadStateRequestWithExpiry {
-  body: {
-    content: Pick<ReadStateRequest, 'request_type' | 'ingress_expiry'>;
-  };
-}
-
-/**
- * Check if an object has a property
- * @param value the object that might have the property
- * @param property the key of property we're looking for
- */
-function hasProperty<O extends object, P extends string>(
-  value: O,
-  property: P,
-): value is O & Record<P, unknown> {
-  return Object.prototype.hasOwnProperty.call(value, property);
-}
-
-/**
- * Check if value is a signed read state request with expiry
- * @param value to check
- */
-function isSignedReadStateRequestWithExpiry(
```

Comment:
I'll actually be using this unused function in my current PR, but it's fine and I can restore it